packages/legend-query-builder/src/components/fetch-structure/QueryBuilderTDSPanel.tsx

```diff
@@ -986,19 +1005,20 @@ export const QueryBuilderTDSPanel = observer(
       [handleDrop],
     );
 
-    const [, projectionColumnDropConnector] = useDrop<
-      QueryBuilderProjectionColumnDragSource,
-      void,
-      { isOverProjectionColumns: boolean }
-    >(
-      () => ({
-        accept: [QUERY_BUILDER_PROJECTION_COLUMN_DND_TYPE],
-        collect: (monitor) => ({
-          isOverProjectionColumns: monitor.isOver({ shallow: false }),
+    const [{ isOverProjectionColumns }, projectionColumnDropConnector] =
```

Comment:
I guess to be clear, let's call these `rearrangeColumnDropConnector` and `isRearrangingColumn`